source/file.go

```diff
@@ -126,11 +126,14 @@ func NewFile(relpath string) *File {
 	f.ext = strings.TrimPrefix(filepath.Ext(f.LogicalName()), ".")
 	f.baseName = helpers.Filename(f.LogicalName())
 
-	f.lang = strings.TrimPrefix(filepath.Ext(f.baseName), ".")
-	if f.lang == "" {
+	lang := strings.TrimPrefix(filepath.Ext(f.baseName), ".")
+	if _, ok := viper.GetStringMap("Languages")[lang]; lang == "" || !ok {
```

Comment:
I took that verbatim from https://gohugo.io/content/multilingual/, I've come to know only now that case of keys does not matter. 😛
I've pushed the improved version.